jpa/src/main/java/org/jdbi/v3/jpa/EntityMemberAccessException.java

```diff
@@ -15,6 +15,7 @@
 
 import org.jdbi.v3.exceptions.DBIException;
 
+@SuppressWarnings("serial")
```

Comment:
I usually just add the `serialVersionUID` field rather than suppressing the warning.